packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -303,6 +307,13 @@ export const Multiple = {
     '[data-testid="connections-list-title-actions-import-saved-connections-action"]',
 
   InUseEncryptionMarker: '[data-action="open-csfle-modal"]',
+
+  ConnectedConnectionItems:
+    '[role="treeitem"][aria-level="1"] [data-is-connected=true]',
+
+  connectionItemByName: (connectionName: string, connected: boolean) => {
```

Comment:
`connectionItemByNameAndStatus`? to prevent confusion in case we'll want one without the status

Reply:
We can also just change it once we need it. This is all internal detail to this one package.

Reply:
Another design might be to turn the second parameter into an options hash and have connected be optional defaulting to undefined. If it is undefined it is igored.